files/windows/dependencies.sh

```diff
@@ -2,6 +2,7 @@
 
 pacman --noconfirm -S --needed \
   git \
+  upx \
```

Comment:
;)

Reply:
This is why I'm happy the Windows packaging process is finally automatic. :)